mathics/builtin/patterns/rules.py

```diff
@@ -0,0 +1,563 @@
+# -*- coding: utf-8 -*-
+"""
+Define, apply and compiling rules.
```

Comment:
Verb order agreement mismatch. Consider: 

_Defining_, _Applyiing_, and compiling